### PR TITLE
Disables ivy as it breaks libraries with ViewEngine compiler

### PR DESCRIPTION
**Pull request by pavankjadda**

### CODE REVIEW

File: projects/ngx-cookie-service/src/lib/cookie.service.ts [hunk 1]
```diff
@@ -12,9 +12,9 @@ export class CookieService {
   private readonly documentIsAccessible: boolean;
 
   constructor(
-    @Inject(DOCUMENT) private document: Document,
+    @Inject(DOCUMENT) private document: any,
```
Comment (stevermeister) [inline projects/ngx-cookie-service/src/lib/cookie.service.ts:15]: ?

File: projects/ngx-cookie-service/tsconfig.lib.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
   "angularCompilerOptions": {
     "skipTemplateCodegen": true,
     "strictMetadataEmit": true,
-    "enableResourceInlining": true
+    "enableResourceInlining": true,
+    "fullTemplateTypeCheck": true
```
Comment (stevermeister) [inline projects/ngx-cookie-service/tsconfig.lib.json:16]: 👍

File: projects/ngx-cookie-service/src/lib/cookie.service.ts (hunk 1 above)
Comment (pavankjadda) [reply projects/ngx-cookie-service/src/lib/cookie.service.ts:15]: Angular team fixed this error with Ivy, so we can use `@Inject(DOCUMENT) private document: Document`

but for users who are using `ViewEngine` it broke the library, issues were caused by this #144  and #145

Comment (stevermeister) [reply projects/ngx-cookie-service/src/lib/cookie.service.ts:15]: Thank you for the explanation!

**Pull request merged**